testsuite/cluster-test/src/main.rs

```diff
@@ -887,18 +948,20 @@ impl ClusterTestRunner {
     }
 
     fn cleanup(&mut self) {
-        let futures = self.cluster.all_instances().map(|instance| async move {
-            RemoveNetworkEffects::new(instance.clone())
-                .apply()
-                .await
-                .map_err(|e| {
-                    info!(
-                        "Failed to remove network effects for {}. Error: {}",
-                        instance, e
-                    );
-                })
-        });
-        self.runtime.block_on(join_all(futures));
+        if self.cluster_swarm.is_none() {
```

Comment:
This should be run in k8s too right? Instances can persist between tests so we need to clean network effects?
(ok to add is as todo to an issue, if you prefer not to address it in this PR)

Reply:
Once we support network effects in k8s, we can add cleanup.